template.yaml

```diff
@@ -172,6 +188,17 @@ Resources:
             Resource:
               - !GetAtt ChannelRegisterCacheTable.Arn
 
+  AppConfigActionsPolicy:
+    Type: AWS::IAM::ManagedPolicy
+    Properties:
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Effect: Allow
+            Action:
+              - appconfig:GetConfiguration
+            Resource: '*'
```

Comment:
https://docs.aws.amazon.com/service-authorization/latest/reference/list_awsappconfig.html#awsappconfig-environment

It seems like AWS does not support resource-level permission for appconfig:GetConfiguration.